src/Microsoft.AspNet.Razor.Runtime/BufferedStringCollection.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Implements <see cref="IBufferedStringCollection"/> to buffer a collection of strings.
+    /// </summary>
+    public class BufferedStringCollection : IBufferedStringCollection
```

Comment:
The idea of this class is to expose only those methods necessary for `TagHelperOutput`. Since MVC (`RazorTextWriter` and `HtmlString`) needs a `TextWriter`, we have a wrapper around this.